packages/design-system/src/components/N8nTabs/Tabs.vue

```diff
@@ -7,27 +7,58 @@
 			<n8n-icon icon="chevron-right" size="small" />
 		</div>
 		<div ref="tabs" :class="$style.tabs">
-			<div  v-for="option in options" :key="option.value" :class="{ [$style.alignRight]: option.align === 'right' }">
-				<a
-					v-if="option.href"
-					target="_blank"
-					:href="option.href"
-					:class="[$style.link, $style.tab]"
-					@click="() => handleTabClick(option.value)"
-				>
-					<div>
-						{{ option.label }}
-						<span :class="$style.external"><n8n-icon icon="external-link-alt" size="small" /></span>
+			<div  v-for="option in options"
+				:key="option.value"
+				:class="{ [$style.alignRight]: option.align === 'right' }"
+			>
+				<n8n-tooltip v-if="option.tooltip" placement="top">
+					<div slot="content">
+						{{ option.tooltip }}
+					</div>
+					<a
```

Comment:
seems like there's a bit of duplication here... there's a disabled `prop` on `n8n-tooltip`.. set that `disabled="!!option.tooltip"` and you don't need the duplication then

Reply:
Nice! 👌